.github/workflows/build-node-wrapper/action.yml

```diff
@@ -65,15 +65,16 @@ runs:
             named_os: ${{ inputs.named_os }}
             arch: ${{ inputs.arch }}
             npm_scope: ${{ inputs.npm_scope }}
-
+            target: ${{ inputs.target }}
+    
         - name: npm install
           shell: bash
           working-directory: ./node
           run: |
               rm -rf node_modules && npm install --frozen-lockfile
               cd rust-client
-              npm install
-
+              npm install --frozen-lockfile
+        
```

Comment:
```suggestion

```

Reply:
It is needed. I know it wasn't there before, but the structure we follow is an empty line between steps.